packages/jest-emotion/src/index.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import { registerEmotionModule } from '@emotion/utils'
```

Comment:
i think this only matters for `@emotion/core` package - don't think there is a much need for this in other packages